src/log.cr

```diff
@@ -85,38 +85,41 @@
 #
 # ```
 # backend = Log::IOBackend.new
+# Log.builder.clear
```

Comment:
Do you have to run `clear` before `bind`?

Reply:
After this PR, if you want to remove the default binds added by setup_from_env, yes you would need that.

Reply:
This seems like a big change and may introduce some weird regressions. If I understand this correctly, if I did this:

```crystal
Log.builder.bind("*", :info, MyCoolbackend.new)
```

The logs would now have 2 backends because it has the default backend + the MyCoolBackend? If so I think this is going to be very confusing for people. I think that setup_from_env should be called explicitly and instead the default should not be "set" but should be something like `Log#severity || :info`. So it does not really on bind at all for it to work and doesn ot require "clearing" the defaults

Reply:
i think it's very important that things gets logged by default. But what about if the "default" configuration gets cleared on the first "manual" bind?

Reply:
I agree a default is great. I think there could be a default that doesn’t have to be “set” from  along seup_from_env though. It could work like most defaults where if nothing is explicitly set it uses a default value